engine/table/src/main/java/io/deephaven/engine/tablelogger/EngineTableLoggerProvider.java

```diff
@@ -0,0 +1,42 @@
+package io.deephaven.engine.tablelogger;
+
+import io.deephaven.engine.tablelogger.impl.memory.EngineTableLoggerMemoryImplFactory;
+
+/**
+ * Provides the factory for providing engine table loggers.
+ */
+public class EngineTableLoggerProvider {
```

Comment:
How about a better class name,  maybe MetricsLoggers so that the usage is less confusing

MetricLoggers.Factory. or EngineLoggers.Provider feel better than EngineTableLoggerProvider.Factory

Reply:
Changed top level to EngineLoggers. Kept nested interface as Factory.

Reply:
Actually, I think I will change it further to EngineTableLoggers for a bit more specificity.